amd_openvx_extensions/amd_rpp/source/Brightness.cpp

```diff
@@ -204,6 +216,10 @@ static vx_status VX_CALLBACK initializeBrightness(vx_node node, const vx_referen
     {
         data->src_desc_ptr->dataType = RpptDataType::F32;
     }
+    else if (data->in_tensor_type == vx_type_e::VX_TYPE_FLOAT16)
+    {
+        data->src_desc_ptr->dataType = RpptDataType::F16;
+    }
```

Comment:
I8 datatype case is not added here. Can we add that case too?
else if (data->in_tensor_type == vx_type_e::VX_TYPE_INT8)
{
data->src_desc_ptr->dataType = RpptDataType::F32data->src_desc_ptr->dataType = RpptDataType::I8;
}